blosc/frame.c

```diff
@@ -847,6 +850,88 @@ blosc2_frame_s* frame_from_file(const char* urlpath, const blosc2_io *io) {
 }
 
 
+/* Initialize a frame out of a file */
+blosc2_frame_s* frame_from_file_offset(const char* urlpath, const blosc2_io *io, int64_t offset) {
```

Comment:
It would be nice if you can remove the `frame_from_file()` function, and replace it by calls to `frame_from_file_offset` with the `offset` param set to 0.

Reply:
Done!